Multisig/Logic/NetworkManager.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ChainManager.swift
+//  Multisig
+//
+//  Created by Moaaz on 6/23/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+class NetworkManager {
```

Comment:
needs unit tests

Reply:
added tests for migrateOldSafes.
I am not sure if we need tests for updateChainsInfo